pkg/controller/clone-controller.go

```diff
@@ -45,112 +56,126 @@ const (
 	// APIServerPublicKeyPath is the path to the apiserver public key
 	APIServerPublicKeyPath = APIServerPublicKeyDir + "/id_rsa.pub"
 
+	// CloneSucceededPVC provides a const to indicate an import to the PVC failed
+	CloneSucceededPVC = "CloneSucceeded"
+
 	cloneSourcePodFinalizer = "cdi.kubevirt.io/cloneSource"
 
 	cloneTokenLeeway = 10 * time.Second
 )
 
-// CloneController represents the CDI Clone Controller
-type CloneController struct {
-	Controller
+// CloneReconciler members
+type CloneReconciler struct {
+	Client         client.Client
+	Scheme         *runtime.Scheme
+	K8sClient      kubernetes.Interface
```

Comment:
not opposed to having these client-go clients around but may want to refactor the code that requires them

Reply:
So my plan is to remove the client-go clients once all the controllers have been converted, and I can refactor any utility functions that use them to use the runtime lib client.